src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/Målgruppe.tsx

```diff
@@ -2,37 +2,30 @@ import React from 'react';
 
 import MålgruppeInfo from './MålgruppeInfo';
 import { Vilkårsregler } from '../../../../typer/regel';
-import { Inngangsvilkår, Vilkår, Vilkårsresultat } from '../../vilkår';
+import { Inngangsvilkårtype, Vilkårsresultat, Vilkårsvurdering } from '../../vilkår';
 import { Vilkårpanel } from '../../Vilkårspanel/Vilkårpanel';
 import { VilkårpanelInnhold } from '../../Vilkårspanel/VilkårpanelInnhold';
 import VisEllerEndreVurdering from '../../Vilkårvurdering/VisEllerEndreVurdering';
 
 interface Props {
-    feilmelding?: string;
-    vilkår: Vilkår;
-    vilkårsregler: Vilkårsregler<Inngangsvilkår.MÅLGRUPPE>;
+    vilkårsregler: Vilkårsregler<Inngangsvilkårtype.MÅLGRUPPE>;
+    vilkårsvurdering: Vilkårsvurdering;
 }
 
-const Målgruppe: React.FC<Props> = ({ feilmelding, vilkår, vilkårsregler }) => {
-    // const vurdering = vurderinger.find(
-    //     (v) => v.vilkårType === InngangsvilkårType.FORUTGÅENDE_MEDLEMSKAP
-    // );
-    // if (!vurdering) {
-    //     return <div>Mangler vurdering for forutgående medlemskap</div>;
-    // }
+const Målgruppe: React.FC<Props> = ({ vilkårsregler, vilkårsvurdering }) => {
+    const vilkår = vilkårsvurdering.vilkårsett.find(
+        (v) => v.vilkårType === Inngangsvilkårtype.MÅLGRUPPE
+    );
+    if (!vilkår) {
+        return <div>Mangler vurdering for forutgående medlemskap</div>;
+    }
```

Comment:
Jeg hadde litt håp tidligere om at denne kunne droppes og at man kunne sendt inn riktig vilkår direkte. Det er litt vanskelig når vilkårsett er en liste og typen ligger i lista. Kunne vi evt. vurdert å gjøre `vilkårsett` til en `Map<VilkårType, VilkårDTO` (eller noe tilsvarende struktur) i stedet for bare `List<VilkårDTO>`

Kan være egen PR for denne endringen om det er aktuelt altså

Reply:
Denne kommentaren gjelder alle steder hvor det gjøres (må gjøres for alle vilkår)

Reply:
Tok en prat, vi har flere vilkår på noen typer